.github/CODEOWNERS

```
@@ -1,9 +1,6 @@
-#owners and reviewers
-cuml_bench/*          @Alexsandruss
-daal4py_bench/*       @Alexsandruss @samir-nasibli
-datasets/*            @Alexsandruss
-modelbuilders_bench/* @Alexsandruss
-report_generator/*    @Alexsandruss
-sklearn_bench/*       @Alexsandruss @samir-nasibli
-xgboost_bench/*       @Alexsandruss
+# owners and reviewers
+configs               @Alexsandruss
+sklbench              @Alexsandruss

```

```suggestion
sklbench              @Alexsandruss @samir-nasibli
```
Is it possible to add people at least for review?